verifier/README.md

```diff
@@ -6,18 +6,30 @@ While [Miden](../miden) crate also contains verifier functionality, if a project
 ## Usage
 This crate exposes a `verify()` function which can be used to verify proofs of program execution. The function takes the following parameters:
 
-* `program_hash: &[u8; 32]` - an array of 32 bytes representing a hash of the program to be verified.
-* `public_inputs: &[u128]` - a list of public inputs against which the program was executed.
-* `outputs: &[u128]` - a list of outputs generated by the program.
-* `proof: &StarkProof` - the proof generated during program execution.
+* `program_hash: Digest` - a hash of the program to be verified (represented as a 32-byte digest).
+* `stack_inputs: &[u64]` - a list of values with the stack was initialized prior to a program's execution.
+* `stack_outputs: &[u64]` - a list of values returned from the stack after a program's execution completes.
```

Comment:
nit: I would probably avoid mixing tenses by rewording this to "a list of the values returned from the stack after the program completed execution."